content/css/concepts/typography/terms/text-decoration/text-decoration.md

```diff
@@ -39,3 +39,11 @@ p {
 ```
 
 Above code indicating that the text in `p` tag will be underlined.
+
+**Output**:
+
+![Output of text styled using CSS in Markdown.](/media/css-typography-decoration-output.png)
+
+**Markdown syntax**:
+
+![CSS code used to style paragraph text in a Markdown document.](/media/css-typography-decoration.png)
```

Comment:
To add images use https://raw.githubusercontent.com/Codecademy/docs/main/media/ followed by the image name.